src/data-service/GameData.js

```diff
@@ -0,0 +1,19 @@
+
+const GameModel = require('../model/GameDto');
+const InitialConnect = require('./SequelizeConnection');
+
+let GameDataController = {};
+
+GameDataController.createGame = async function (newGameData) {
+    return new Promise((resolve, reject) => {
+      let tempConnect = InitialConnect();      
```

Comment:
the connection to the database should be done only once when the server is starting, with the current implementation you are creating a new connection everytime you create a game and this is expensive in terms of performance

Reply:
The initialization was token globally